site/public/js/notification-sound.js

```diff
@@ -0,0 +1,19 @@
+        }
```

Comment:
Why include both, and not just wav? Comparing support for [wav](https://caniuse.com/?search=wav%20audio%20format) vs [ogg](https://caniuse.com/?search=ogg%20audio%20format), it looks like wav supports all the same platforms as ogg with the addition of Safari.

Reply:
Good catch, I was looking at an outdated table when I was trying to decide what audio types to include.  I will rework this to only check and include wav.

Reply:
Looking at the docs page for [canPlayType](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/canPlayType), we might want the conditional here to also be just `this.sound.canPlayType('audio/wav') !== ''` as this function can return `'probably'`, `'maybe'`, and `''`, even though in practice `'maybe'` and `''` are the only ones returned.

Reply:
Reworked this